Code.Rmd

```diff
@@ -29,23 +42,126 @@ Congratulations!
 You've learned your first package development workflow.
 Even if you learn nothing else from this book, you'll have gained a useful workflow for editing and reloading R code.
 
-## Organising your functions
+## Organising your functions into files {#r-organising}
+
+*We originally decided to remove this in deference to material in <https://style.tidyverse.org>. The removed text is in [tidyverse/style/#121](https://github.com/tidyverse/style/issues/121). In hindsight, I think removal is too extreme and I've re-introduced this section. First, the original material really isn't in style.tidyverse.org (yet?). Second, people really want guidance on this and it feels misguided to omit from this book out of some adherence to the DRY principle.*
+
+The only hard rule is that your package should store its function definitions in R scripts, i.e. files with extension `.R`, that live in the `R/` directory[^no-subdirs].
```

Comment:
I think it's worth making the distinction that R doesn't care, but human readers will.

Reply:
I feel like that's what the very next sentence does 👇 It's certainly the intent